lenskit-eval/src/main/java/org/lenskit/eval/traintest/recommend/RecommendEvalTask.java

```diff
@@ -258,6 +303,10 @@ public void addMetric(TopNMetric<?> metric) {
 
     @Override
     public List<String> getUserColumns() {
+        if (separateItems) {
+            Collections.emptyList();
```

Comment:
Just so I'm clear, we do this because we use a different file for the seperated (1r) evaluations. right?